petrel/setup.py

```diff
@@ -13,12 +13,12 @@
 
 PACKAGE = "petrel"
 
-PETREL_VERSION = '0.3'
+PETREL_VERSION = '0.10.0'
```

Comment:
Petrel version isn't supposed to match the Storm version. The Storm version makes up the last 3 parts of the overall version, e.g. 0.3.0.10.0.